libminifi/include/core/ContentRepository.h

```diff
@@ -53,10 +54,18 @@ class ContentRepository : public StreamManager<minifi::ResourceClaim>, public ut
 
   virtual void clearOrphans() = 0;
 
+  bool remove(const minifi::ResourceClaim &streamId) override {
```

Comment:
can this be `final`?

Reply:
Updated in bde565c405e3e9419020445a48d41da561b58729